scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -283,6 +286,17 @@ task ExtractTask {
              -O interval-files \
              ~{split_intervals_extra_args}
          cp interval-files/*.interval_list .
+
+         # Drop trailing slash if one exists
+         OUTPUT_GCS_DIR=$(echo ~{output_gcs_dir} | sed 's/\/$//')
+
+         if [ -n "$OUTPUT_GCS_DIR" ]; then
+             if [ ~{has_service_account_file} = 'true' ]; then
+                 gsutil cp ~{service_account_json_path} local.service_account.json
+                 gcloud auth activate-service-account --key-file=local.service_account.json
+             fi
+             gsutil -m cp *.interval_list $OUTPUT_GCS_DIR/
```

Comment:
any organizational benefit to putting it into an interval list specific directory?

Reply:
🤷🏻‍♀️